dash_docs/chapters/dash_enterprise/dash_enterprise_chapters.py

```diff
@@ -1988,44 +1952,33 @@ def display_instructions(platform):
     ''')
     ]),
 
-
-
-    rc.Markdown('''
-
-    #### Service Linking Commands:
-
-    > These commands can only be run by the user who owns both the service as well as the application.
-    '''),
-
-    html.Details([
-        html.Summary("Link Redis to an app"),
-        rc.Markdown('''
-        &nbsp;
-
-        Link the Redis service to the app. This will also restart your app:
-
-        **Example:**
-
-        `ssh dokku@your-dash-enterprise -p PORT redis:link redis-db my-dash-app`
-
-        &nbsp;
-    ''')
+    dcc.Markdown([
+        '''
+        ### Unsupported and Deprecated Commands
+        
+        These commands are no longer supported or recommended.
```

Comment:
```suggestion
        These `dokku` commands are no longer supported or recommended. Each of their recommended alternatives, respectively, are provided below.
```

Non-blocking, I'm just thinking that a user might see "unsupported and deprecated" and skip the entire section, or get confused and think all of the below commands are deprecated, at least until they read enough to get the context.